index.php

```diff
@@ -45,4 +46,9 @@
 $PageTitle .= $Config['SiteName'];
 $PageMetaDesc = htmlspecialchars(mb_substr($Config['SiteDesc'], 0, 150, 'utf-8'));
 $ContentFile  = $TemplatePath . 'home.php';
-include($TemplatePath . 'layout.php');
\ No newline at end of file
+if (file_exists($TemplatePath . 'layout.php')) {

```

Comment:
Check if the template has been setup correctly because this step is very important.